ext/reflection/php_reflection.c

```diff
@@ -6064,11 +6062,13 @@ ZEND_METHOD(ReflectionProperty, setValue)
 
 		zend_update_static_property_ex(intern->ce, ref->unmangled_name, value);
 	} else {
-		if (zend_parse_parameters(ZEND_NUM_ARGS(), "oz", &object, &value) == FAILURE) {
-			RETURN_THROWS();
-		}
+		zend_object *object;
+		ZEND_PARSE_PARAMETERS_START(2, 2) {
```

Comment:
We normally don't use `{` and `}` around the fast-ZPP macros. I've never seen it in php-src at least.

Reply:
I've stolen that from `setRawValueWithoutLazyInitialization`:

https://github.com/php/php-src/blob/41996e8d4fffff5b027b45e599e5217a470f723c/ext/reflection/php_reflection.c#L6198-L6201

But indeed that function and `skipLazyInitialization` are the only two functions that use it like this.

Reply:
Even php code style consistently inconsistent.
Anyway, let's drop the braces in your PR and then we can drop the braces from those functions as well in a follow-up.